client-spark/common/src/main/java/org/apache/uniffle/shuffle/manager/ShuffleManagerServerFactory.java

```diff
@@ -43,7 +43,7 @@ public GrpcServer getServer() {
 
   public GrpcServer getServer(ShuffleManagerGrpcService service) {
     ServerType type = conf.get(RssBaseConf.RPC_SERVER_TYPE);
-    if (type == ServerType.GRPC) {
+    if (type == ServerType.GRPC || type == ServerType.GRPC_NETTY) {
```

Comment:
This does not look like fixing flaky tests but like a feature + testing it. Maybe the PR title should be rephrased.